services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -427,51 +427,69 @@ open class ApplicationService(
         }
     }
 
-    private fun sendApplicationToAllu(application: ApplicationEntity): Int {
+    private fun sendToAllu(entity: ApplicationEntity): Int {
```

Comment:
What was wrong with saying what was being sent in the method name?

Reply:
I think it is pretty clear from the function signature what will be sent. But I will change it back to what it was.

Reply:
Fixed.